roles/pulibrary.apache_tomcat/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ tomcat_major_version: "{{ tomcat_version.split('.')[0].tomcat_version.split('.')
 
 tomcat_install_java: true
 tomcat_java_version: 1.8
-tomcat_java_home: /usr/lib/jvm/jre
+tomcat_java_home: /usr/lib/jvm/java-8-openjdk-amd64/jre
```

Comment:
https://github.com/pulibrary/princeton_ansible/pull/1624/files#diff-98b8cd6f145948b908467d0d4a93b9cfR10 refers explicitly to the `jdk`, but this links to the `jre` - for my own understanding, is this just more efficient for running the `tomcat` process?

Reply:
noted and will be in dataspace PR

Reply:
if we don't use jdk... ant fails to build dspace so the jdk is there for that. 
Also while creating the new pulibrary.apache_tomcat role I had to choose if I wanted to also do a manual install of openjdk and felt it was overkill.